src/routes/album.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const albumController = require('../controllers/album');
+
+const router = express.Router();
+
+router.post('/:id/albums/', albumController.createAlbum);
+
+router.get('/', albumController.readAlbum);
+
+router.get('/:id', albumController.getAlbumById);
```

Comment:
You could bunch these two together if not all of these. Theres different ways of looking at it but generally you want to group the statements according to their verb if nothing else
```
router.get('/', albumController.readAlbum);
router.get('/:id', albumController.getAlbumById);
```